packages/backend/src/services/InstanceConfigurationService/InstanceConfigurationService.ts

```diff
@@ -286,4 +254,164 @@ export class InstanceConfigurationService extends BaseService {
             throw error;
         }
     }
+
+    async updateInstanceConfiguration() {
```

Comment:
Could this method be broken up into additional methods to group the operations being performed?

Reply:
I created different private methods inside the service